libseccomp/src/lib.rs

```diff
@@ -639,6 +690,16 @@ impl ScmpFilterContext {
     ///
     /// If this function is called with an invalid filter or an issue is
     /// encountered retrieving the attribute, an error will be returned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use libseccomp::*;
+    /// let mut ctx = ScmpFilterContext::new_filter(ScmpAction::Allow)?;
+    /// ctx.reset(ScmpAction::Allow)?;
+    /// let action = ctx.get_filter_attr(ScmpFilterAttr::ActDefault)?;
```

Comment:
Could fix it as follows?

```rust
let action = ctx.get_filter_attr(ScmpFilterAttr::ActDefault)?;
assert_eq!(ScmpAction::Allow, action);
```

Reply:
Dear @ManaSugi san,

Modified as below:

```
let mut ctx = ScmpFilterContext::new_filter(ScmpAction::Allow)?;
assert_eq!(ctx.get_filter_attr(ScmpFilterAttr::CtlNnp)?,1);
```

Please review.